src/shared/components/common/post-hidden-select.tsx

```diff
@@ -27,40 +27,41 @@ export default class PostHiddenSelect extends Component<
         className="show-hidden-select btn-group btn-group-toggle flex-wrap"
         role="group"
       >
+        <input
+          id="show-hidden"
+          type="radio"
+          className="btn-check"
+          value="true"
+          checked={showHidden === "true"}
+          onChange={linkEvent(this, handleShowHiddenChange)}
+        />
         <label
           htmlFor="show-hidden"
           className={classNames("pointer btn btn-outline-secondary", {
             active: showHidden === "true",
           })}
           data-tippy-content={I18NextService.i18n.t("show_hidden_posts")}
+          aria-label="Show hidden posts"
```

Comment:
Use `I18NextService.i18n.t("show_hidden_posts")` . Otherwise it won't be internationalized.